src/redux/users/usersSlice.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable */
+// import { createSlice } from "@reduxjs/toolkit";
+// import { createAsyncThunk } from "@reduxjs/toolkit";
+
+// const initialState = {
+//     loading: false,
+//     users: [],
+//     error: ''
+// }
+
+// const UserSlice = createSlice({
+//     name: 'user',
+//     initialState,
+// })
+
+const reciver = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/ZEAeqeXuXcX9bQfaMnAJ/books'
+const getId = async () => {
+    const options = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    };
+    const response = await fetch(reciver, options);
+    const result = await response.text();
+    console.log(result)
+  };
+  //ZEAeqeXuXcX9bQfaMnAJ
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1:

Reply:
- [ ] Please remove this commented piece of code as it clutters your codebase and is a nightmare for anyone reading your code 👍